Example/Pods/Target Support Files/Pods-NativeMessageExample/Pods-NativeMessageExample.debug.xcconfig

```diff
@@ -1,11 +1,10 @@
 ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
 CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO
-FRAMEWORK_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/ConsentViewController-iOS" "${PODS_CONFIGURATION_BUILD_DIR}/Wormholy"
+FRAMEWORK_SEARCH_PATHS = $(inherited) "$(PLATFORM_DIR)/Developer/Library/Frameworks" "${PODS_CONFIGURATION_BUILD_DIR}/ConsentViewController-iOS" "${PODS_CONFIGURATION_BUILD_DIR}/Nimble-iOS" "${PODS_CONFIGURATION_BUILD_DIR}/Quick-iOS" "${PODS_CONFIGURATION_BUILD_DIR}/Wormholy"
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
-HEADER_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/ConsentViewController-iOS/ConsentViewController.framework/Headers" "${PODS_CONFIGURATION_BUILD_DIR}/Wormholy/Wormholy.framework/Headers"
-LD_RUNPATH_SEARCH_PATHS = $(inherited) /usr/lib/swift '@executable_path/Frameworks' '@loader_path/Frameworks'
-LIBRARY_SEARCH_PATHS = $(inherited) "${DT_TOOLCHAIN_DIR}/usr/lib/swift/${PLATFORM_NAME}" /usr/lib/swift
-OTHER_LDFLAGS = $(inherited) -framework "ConsentViewController" -framework "Foundation" -framework "Wormholy"
+HEADER_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/ConsentViewController-iOS/ConsentViewController.framework/Headers" "${PODS_CONFIGURATION_BUILD_DIR}/Nimble-iOS/Nimble.framework/Headers" "${PODS_CONFIGURATION_BUILD_DIR}/Quick-iOS/Quick.framework/Headers" "${PODS_CONFIGURATION_BUILD_DIR}/Wormholy/Wormholy.framework/Headers"
+LD_RUNPATH_SEARCH_PATHS = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
+OTHER_LDFLAGS = $(inherited) -framework "ConsentViewController" -framework "Foundation" -framework "Nimble" -framework "Quick" -framework "Wormholy" -framework "XCTest" -weak_framework "XCTest"
```

Comment:
May it be related to the not passing `NativeMessageUITests` suite? I've tried revert back to `LD_RUNPATH_SEARCH_PATHS = $(inherited) /usr/lib/swift` & no luck as result

Reply:
Adding `LIBRARY_SEARCH_PATHS = $(inherited) "${DT_TOOLCHAIN_DIR}/usr/lib/swift/${PLATFORM_NAME}" /usr/lib/swift` is not working either